src/Pages/About/Hero2/Hero2.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Searchnav from '../../../components/Searchnav/Searchnav';
+import Monkey from '../../../asserts/images/monkey.png';
+import icon from '../../../asserts/images/icon.png';
+import './Hero2.css';
+
+export default function Hero2() {
```

Comment:
Hero2 is not an appropriate name for a react component. Make the name more meaningful. Is it not AboutPageHeroSection?